TAScheduler/AcceptanceTests/admin_create_course_test.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+from django.urls import reverse
+from TAScheduler.models import User, Course, Instructor, TA, Administrator
+
+
+class AdminCreateCourseTestCase(TestCase):
+    def setUp(self):
+        # Create an admin user for login
+        admin_user = User.objects.create(
+            email_address='admin@example.com',
+            password='adminpassword',
+            first_name='Admin',
+            last_name='User',
+            home_address='123 Admin St',
+            phone_number='1234567890'
+        )
+        Administrator.objects.create(user=admin_user)
+        self.client.login(email_address=admin_user.email_address,
+                          password='adminpassword')  # Update as per your login logic
+
+    def test_create_course_success(self):
+        # Test that admin can create a new course successfully
+        response = self.client.post(reverse('create_course'), {
+            'course_id': 101,
+            'semester': 'Fall 2023',
+            'name': 'Introduction to Testing',
+            'description': 'A course about writing tests in Django.',
+            'num_of_sections': 3,
+            'modality': 'Online',
+            'credits': 4
+        })
+        self.assertEqual(response.status_code, 302)  # Assuming redirection after successful creation
+        self.assertTrue(Course.objects.filter(name='Introduction to Testing').exists())
+
+    def test_create_course_duplicate(self):
+        # Create a course first
```

Comment:
maybe move this to the setup to make the test easier to read? (I am horrible at making tests succinct but I find doing this makes it a little easier to understand the test)